app/views/rails_admin/main/_show_hide_js.html.haml

```diff
@@ -1,11 +1,38 @@
 :javascript
   $(document).ready(function() {
-    document.getElementById('game_prizes_text_field').style.display = "none";
-    document.getElementById('game_completion_certificate_template_field').style.display = "none";
+    // Hide toggleable fields (Game)
+    try {
+      document.getElementById('game_prizes_text_field').style.display = "none";
+      document.getElementById('game_completion_certificate_template_field').style.display = "none";
+    } catch {}
 
-    document.getElementById('game_enable_completion_certificates').addEventListener('change', toggleDisplayCert);
-    document.getElementById('game_prizes_available').addEventListener('change', toggleDisplayPrizes);
+    // Toggle sponsor fields based on if the checkmark is checked
+    try {
+      toggleSponsorField()
```

Comment:
Instead of a try/catch block is it possible to just check if the elements exist on the page and use that information? Try/catch seems a little heavy handed here.